common/containers/Tabs/Swap/components/PartThree.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import type {
+  LoadBityRatesRequestedSwapAction,
+  RestartSwapAction,
+  StopLoadBityRatesSwapAction
+} from 'actions/swap';
+import SwapProgress from './SwapProgress';
+import PaymentInfo from './PaymentInfo';
+
+type ReduxStateProps = {
+  destinationAddress: string,
+  destinationKind: string,
+  originKind: string,
+  originAmount: ?number,
+  destinationAmount: ?number,
+  isPostingOrder: boolean,
+  reference: string,
+  secondsRemaining: ?number,
+  paymentAddress: ?string,
+  orderStatus: ?string
+};
+
+type ReduxActionProps = {
+  loadBityRatesRequestedSwap: () => LoadBityRatesRequestedSwapAction,
+  restartSwap: () => RestartSwapAction,
+  stopLoadBityRatesSwap: () => StopLoadBityRatesSwapAction,
+  startOrderTimerSwap: Function,
+  startPollBityOrderStatus: Function,
+  stopOrderTimerSwap: Function,
+  stopPollBityOrderStatus: Function,
+  showNotification: Function
+};
+
+export default class PartThree extends Component {
```

Comment:
Is there a more descriptive name for what this does? It might be awkward if we add another step before it, and it's a little tough to tell what it covers without cracking open the code yourself.